css/properties/box-shadow.json

```diff
@@ -0,0 +1,318 @@
+{
+  "css": {
+    "properties": {
+      "box-shadow": {
```

Comment:
I think there are some comments missing here.

Reply:
Fixed. Must've opened the PR before I had completely finished there. Sorry!